<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,16,1">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,2,20,1">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>智能分拣系统</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,2">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <widget class="QLabel" name="label_image">
             <property name="geometry">
              <rect>
               <x>350</x>
               <y>0</y>
               <width>317</width>
               <height>256</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QWidget" name="widget_view" native="true">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>317</width>
               <height>265</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="8,1,8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_SerialConnnect">
                <property name="text">
                 <string>连接串口</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>speed</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QTextBrowser" name="textBrowser">
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(255, 254, 249, 125);</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>48</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="placeholderText">
                   <string>区域编号</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_Getdata">
                  <property name="text">
                   <string>抓取</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>选择摄像头</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_opencamera">
                  <property name="text">
                   <string>打开摄像头</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_auto">
                  <property name="text">
                   <string>自动抓取</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButton_Insert">
                  <property name="text">
                   <string>记录</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>舵机0</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox">
                    <property name="minimum">
                     <number>1000</number>
                    </property>
                    <property name="maximum">
                     <number>2000</number>
                    </property>
                    <property name="singleStep">
                     <number>20</number>
                    </property>
                    <property name="value">
                     <number>1500</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>舵机1</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_2">
                    <property name="minimum">
                     <number>1000</number>
                    </property>
                    <property name="maximum">
                     <number>2000</number>
                    </property>
                    <property name="singleStep">
                     <number>20</number>
                    </property>
                    <property name="value">
                     <number>1500</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>舵机2</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_6">
                    <property name="minimum">
                     <number>1000</number>
                    </property>
                    <property name="maximum">
                     <number>2000</number>
                    </property>
                    <property name="singleStep">
                     <number>20</number>
                    </property>
                    <property name="value">
                     <number>1500</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>舵机3</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_4">
                    <property name="minimum">
                     <number>1000</number>
                    </property>
                    <property name="maximum">
                     <number>2000</number>
                    </property>
                    <property name="singleStep">
                     <number>20</number>
                    </property>
                    <property name="value">
                     <number>1500</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>舵机4</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_3">
                    <property name="minimum">
                     <number>1000</number>
                    </property>
                    <property name="maximum">
                     <number>2000</number>
                    </property>
                    <property name="singleStep">
                     <number>20</number>
                    </property>
                    <property name="value">
                     <number>1500</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>舵机5</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_5">
                    <property name="minimum">
                     <number>1000</number>
                    </property>
                    <property name="maximum">
                     <number>2000</number>
                    </property>
                    <property name="singleStep">
                     <number>20</number>
                    </property>
                    <property name="value">
                     <number>1500</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
